src/workspaces/ShareWorkspaceModal/Collaborator.tsx

```diff
@@ -22,12 +22,13 @@ interface CollaboratorProps {
   acl: WorkspaceAcl;
   setAcl: Dispatch<SetStateAction<WorkspaceAcl>>;
   originalAcl: WorkspaceAcl;
-  workspace: BaseWorkspace;
+  workspaceAccessLevel: WorkspaceAccessLevel;
```

Comment:
These are the only parts of `workspace` that were used in the `Collaborator` component, so only requiring them makes it possible to use the component without needing an entire pre-existing workspace